### DISCUSSION TITLE
Tuta Calendar as a separated app (all platforms)

### DISCUSSION THREAD

**Discussion by serrq** [tutao/tutanota#7291] [Ideas]
Fully independent but also fully integrated, if needed, with all other Tuta ecosystem apps.

I noticed that launching Tuta Mail, after go to calendar it not very comfortable and quick.

I tend to use other apps specifically developed for calendar event, because I want a direct approach to my calendar.

Of course, once it has a dedicated app, it must be removed from Tuta Mail app.

**Comment by rezbyte** [comment]
Hi @serrq,
Thank you for your suggestion.

Separating the calendar into it's own app is planned for the mobile platforms, but not for the other platforms (desktop and web) currently.
